code/modules/antagonists/heretic/knowledge/rust_lore.dm

```diff
@@ -95,56 +95,10 @@
 	route = PATH_RUST
 
 /datum/heretic_knowledge/rust_regen/on_gain(mob/user, datum/antagonist/heretic/our_heretic)
-	RegisterSignal(user, COMSIG_MOVABLE_MOVED, PROC_REF(on_move))
-	RegisterSignal(user, COMSIG_LIVING_LIFE, PROC_REF(on_life))
+	user.AddElement(/datum/element/leeching_walk)
```

Comment:
Hey this seems entirely un-related to cult vs heretic.
Should it not be an atomized change?